src/BlockTemplatesController.php

```diff
@@ -118,7 +118,9 @@ public function get_single_block_template( $template, $id, $template_type ) {
 		}
 
 		$available_templates = $this->get_block_templates( array( $slug ) );
-		return ( is_array( $available_templates ) && count( $available_templates ) > 0 ) ? (object) $available_templates[0] : $template;
+		return ( is_array( $available_templates ) && count( $available_templates ) > 0 )
```

Comment:
We've removed the aspect of the code which casted the return value to `(object)`, do you think we should put this back in?

Reply:
@tjcafferkey I remove it because `BlockTemplateUtils::gutenberg_build_template_result_from_file` always return a `WP_Block_Template` object, so it's not necessary to cast the value